.github/workflows/cross-chain-arbitrum.yml

```diff
@@ -0,0 +1,181 @@
+name: Cross-chain Arbitrum module
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - main
+    paths:
+      - "cross-chain/arbitrum/**"
+      - ".github/workflows/cross-chain-arbitrum.yml"
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment (network) for workflow execution, e.g. `goerli`"
+        required: false
+      upstream_builds:
+        description: "Upstream builds"
+        required: false
+      upstream_ref:
+        description: "Git reference to checkout (e.g. branch name)"
+        required: false
+        default: "main"
```

Comment:
Let's remove the `workflow_dispatch:` as this is a part of the CI process jobs that we decided to extract from this PR.

```suggestion
```

Reply:
Removed